integration/storagetest/maptests.go

```diff
@@ -256,3 +256,45 @@ func (*mapTests) TestSetGetRoundTrip(ctx context.Context, t *testing.T, ms stora
 		}
 	}
 }
+
+func (*mapTests) TestLatestSignedMapRoot(ctx context.Context, t *testing.T, ms storage.MapStorage, as storage.AdminStorage) {
+	tree := mustCreateTree(ctx, t, as, storageto.MapTree)
+	mustSignAndStoreMapRoot(ctx, t, ms, tree, &types.MapRootV1{Revision: uint64(0)})
+	treeRev := int64(1) // After tree initialization, the first revision is 1.
+
+	timestamp := time.Now() // ensure the heads we create are newer than any initial head.
+	for i := int64(0); i < 10; i++ {
+		t.Logf("Starting iteration %d", i)
+		meta := []byte{byte(i)}
+
+		sth := &types.MapRootV1{
```

Comment:
```suggestion
		root := &types.MapRootV1{
```
STH is log-specific at the mo.

Reply:
done